pip/commands/cache.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import
+
+import logging
+import os
+
+from pip.basecommand import Command
+from pip.status_codes import SUCCESS, ERROR
+
+
+logger = logging.getLogger(__name__)
+
+
+class CacheCommand(Command):
+    """\
+    Show information about pip's wheel cache.
+
+    Subcommands:
+        location: Print the location of the wheel cache."""
+    actions = ["location"]
+    name = "cache"
+    usage = """
+      %%prog [options] %s""" % "|".join(actions)
+    summary = "Show information about pip's wheel cache."
+
+    def run(self, options, args):
+        if not args or args[0] not in self.actions:
+            logger.warning(
+                "ERROR: Please provide one of these subcommands: %s" %
+                ", ".join(self.actions))
+            return ERROR
+        method = self.__getattribute__("action_%s" % args[0])
+        return method(options, args[1:])
+
+    def action_location(self, options, args):
+        logger.info(os.path.join(options.cache_dir, "wheels"))
```

Comment:
I ask because I'm trying to think of a use case where you don't need to be able to explicitly ask for http versus wheel cache OR bake in an assumption about the names of those caches relative to the overall cache directory that isn't ``rm -rf $(pip cache location)`` or ``du -h -d 0 $(pip cache location)``.